event-loop.c

```diff
@@ -10,15 +10,77 @@
 
 #include "kanshi.h"
 
+static int set_pipe_flags(int fd) {
+	int flags = fcntl(fd, F_GETFL);
+	if (flags == -1) {
+		fprintf(stderr, "fnctl F_GETFL failed: %s\n", strerror(errno));
```

Comment:
This can be simplified with `perror`. Same for all of the error messages below.

Reply:
Oof, I completely missed that. Thanks